core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -75,8 +75,8 @@ sealed trait ZValidation[+W, +E, +A] { self =>
    */
   final def flatMap[W1 >: W, E1 >: E, B](f: A => ZValidation[W1, E1, B]): ZValidation[W1, E1, B] =
     self match {
-      case Failure(w, e) => Failure(w, e)
-      case Success(w, a) =>
+      case failure: Failure[_, _] => failure
+      case Success(w, a)          =>
```

Comment:
Very minor (feel free to ignore) but I generally prefer to put the `Success` cases first (happy path).

Reply:
Yeah I hesitated to change but in the end I preferred to minimize the changes